crates/ruff_python_literal/src/format.rs

```diff
@@ -344,7 +417,7 @@ pub enum FormatParseError {
     UnmatchedBracket,
     MissingStartBracket,
     UnescapedStartBracketInLiteral,
-    InvalidFormatSpecifier,
+    PlaceholderRecursionExceeded,
```

Comment:
The former was only used to represent the latter which I found confusing and unnecessarily vague.